spec/dummy/config/application.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'boot'
+
+# Pick the frameworks you want:
+# require "active_record/railtie"
+require "action_controller/railtie"
+require "action_view/railtie"
+# require "action_mailer/railtie"
+require "active_job/railtie"
```

Comment:
Remove this line please – front-end websites shouldn't need or be using ActiveJob